lib/rubygems/commands/sign_command.rb

```diff
@@ -23,6 +24,8 @@ def usage # :nodoc:
   end
 
   def execute
+    priv, pub = Crypto.new().generate_keys
```

Comment:
Do you do anything with the resulting certificate yet? Or is that still in progress?

Reply:
Still in progress, just trying to feed things in.